torchqdynamics/operators.py

```diff
@@ -21,54 +22,45 @@
 ]
 
 
-def sigmax(
-    *, dtype: torch.dtype = torch.complex128, device: torch.device | None = None
-) -> Tensor:
+@complex_tensor
+def sigmax(*, dtype=None, device=None) -> Tensor:
```

Comment:
Is it not needed to use typing anymore here? Because of the decorator?

Reply:
It's never needed to use typing 😋 I think we can afford to be a bit loose with the typing here, because it's a bit technical for not much. Generally the two main advantages of typing the code is (1) for static type checking (e.g. with mypy) and (2) for readability (for users and developers). Here I don't think we need (1), and (2) is rather harmed than helped by the typing so I'd say it's better to remove it.